subprojects/rzwinkd/winkd.c

```diff
@@ -1394,3 +1392,93 @@ int winkd_break_read(WindCtx *ctx) {
 #endif
 	return 1;
 }
+
+static inline char *download_pdb(const char *path, const char *symserver, const char *symstore) {
+	rz_sys_setenv("RZ_BIN_PDBSERVER", symserver);
+	rz_sys_setenv("RZ_BIN_SYMSTORE", symstore);
+	char *direscaped = rz_str_escape_sh(path);
+	char *ret = rz_sys_cmd_strf("rz-bin -PPj %s", direscaped);
```

Comment:
Same, is it possible to use the API? cc @ret2libc

Reply:
is `rz_bin_pdb_download` the API?